app/src/components/fileTypes.js

```diff
@@ -0,0 +1,28 @@
+const log = require('npmlog');
+
+// manually updated dictionary of input/output file types supported by the carbone js library
+const fileTypesDictionary = {
+  "csv":["doc", "docx", "html", "odt", "pdf", "rtf", "txt", "csv"],
+  "docx":["doc", "docx", "html", "odt", "pdf", "rtf", "txt"],
+  "html":["html", "odt", "pdf", "rtf", "txt"],
+  "odt":["doc", "docx", "html", "odt", "pdf", "rtf", "txt"],
+  "pptx":["odt", "pdf"],
+  "rtf":["docx","pdf"],
+  "txt":["doc", "docx", "html", "odt", "pdf", "rtf", "txt"],
+  "xlsx":["odt", "pdf", "rtf", "txt", "csv", "xls", "xlsx"]
+};
```

Comment:
Style: there needs to be a space between the colon and the open brace. Assuming your IDE is set up correctly it should be able to auto-format that for you.